4-perftools/Dockerfile

```diff
@@ -33,7 +32,7 @@ USER 0
 
 WORKDIR ${HOME}
 
-COPY contrib/etc/scl_enable /root/etc/scl_enable
+COPY root/root/etc/scl_enable /root/etc/scl_enable
```

Comment:
The first `root` is the directory we have in this git repo that contains all the files that should be copied to `/`. And the second `root` is because the contents of the previous `contrib` directory were actually copied into `/root` (see above).